src/ui/TypingIndicatorMessage/index.tsx

```diff
@@ -1,44 +1,98 @@
+const AVATAR_DIAMETER_WITHOUT_BORDER = 24;
+const AVATAR_DIAMETER = AVATAR_DIAMETER_WITHOUT_BORDER + (AVATAR_BORDER_SIZE * 2);
+const LEFT_GAP = 20;
+
+export interface AvatarStackProps {
+  sources: string[];
+  max: number;
+}
+
+const AvatarStack = (props: AvatarStackProps) => {
+  const { sources, max } = props;
 
   return (
-    <div
-      className='sendbird-message-content incoming'
-      style={{ marginBottom: '2px' }}
-    >
-      <div className='sendbird-message-content__left incoming'>
+    <> {
+      sources.slice(0, max).map((src, index) => (
         <Avatar
           className={'sendbird-message-content__left__avatar'}
-          src={typingMember.profileUrl || ''}
+          src={src || ''}
+          key={`avatar_stack_item_${src}`}
           // TODO: Divide getting profileUrl logic to utils
-          width="28px"
-          height="28px"
+          width={`${AVATAR_DIAMETER_WITHOUT_BORDER}px`}
+          height={`${AVATAR_DIAMETER_WITHOUT_BORDER}px`}
+          zIndex={index}
+          left={`${index * LEFT_GAP}px`}
+          border={`${AVATAR_BORDER_SIZE}px solid white`}
         />
-      </div>
-
-      <div className='sendbird-message-content__middle'>
-        <Label
-          className="sendbird-message-content__middle__sender-name"
-          type={LabelTypography.CAPTION_2}
-          color={LabelColors.ONBACKGROUND_2}
-        >
-          {`${typingMember.nickname} ${stringSet.TYPING_INDICATOR__IS_TYPING}`}
-        </Label>
-        <TypingDots/>
-      </div>
+      ))
+    } </>
+  );
+};
+
+const TypingIndicatorMessageAvatar = (props : TypingIndicatorMessageProps) => {
+  const { typingMembers } = props;
+  const membersCount = typingMembers.length;
+  const displayCount = Math.min(membersCount, 4);
+  const hiddenCount = membersCount - 3;
+  const superImposedWidth = ((displayCount - 1) * (AVATAR_DIAMETER - LEFT_GAP));
+  const rightPaddingSize = 12;
+
+  return (
+    <div
+      className='sendbird-message-content__left incoming'
+      style={{
+        minWidth: (displayCount * AVATAR_DIAMETER) - superImposedWidth + rightPaddingSize,
+      }}
+    >
+      <AvatarStack
+        sources={typingMembers.map((member) => member.profileUrl)}
+        max={3}
+      />
+      {
+        hiddenCount > 0
+          ? <Avatar
+            className={'sendbird-message-content__left__avatar'}
+            // TODO: Divide getting profileUrl logic to utils
+            width={`${AVATAR_DIAMETER_WITHOUT_BORDER}px`}
+            height={`${AVATAR_DIAMETER_WITHOUT_BORDER}px`}
+            zIndex={3}
+            left={`${3 * LEFT_GAP}px`}
+            border={`${AVATAR_BORDER_SIZE}px solid white`}
+            customDefaultComponent={({ width, height }) => (
+              <AvatarDefault width={width} height={height} text={`+${hiddenCount}`} />
+            )}
+          />
+          : null
+      }
     </div>
   );
-}
+};
+
+const TypingIndicatorMessage = (props : TypingIndicatorMessageProps) => {
```

Comment:
```suggestion
const TypingIndicatorMessage = (props: TypingIndicatorMessageProps) => {
```

Reply:
In the case of code style mistakes, there is typically a possibility of the same mistake existing in multiple places. It's better to review the entire code rather than just fixing the reported issue in one place.

Reply:
Actually i think this is done by `yarn line:fix`